jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css

```diff
@@ -0,0 +1,163 @@
+.splitScreen {
+    height: calc(100vh);
+    display: flex;
+    flex-direction: row;   
+}
+
+.leftSide {    
+    width: 50%;    
+    display: flex;    
+    align-items: center;
+    justify-content: center;
+    background-color: #023C47;
+}
+
+.loginRectangle {
+    width: 505px;
+    height: 701px;
+    top: 216px;
+    left: 125px;
+    border-radius: 10px;
+    border: 0.5px;
+    background: linear-gradient(0deg, #FFFFFF, #FFFFFF);
+    border: 0.5px solid #878787
+}
+
+.rightSide {    
+    width: 50%;    
+    display: flex;    
+    align-items: center;
+    justify-content: center;
+}
+
+.rightImage {
+    width: 50%;
```

Comment:
Can you update to 60% for height/width?

Reply:
And do the same for the other page as well

Reply:
done